Tropos/Controllers/TRApplicationController.m

```diff
@@ -58,4 +58,20 @@ - (void)setMinimumBackgroundFetchIntervalForApplication:(UIApplication *)applica
     }
 }
 
+- (void)subscribeForNotificationsWithDeviceToken:(NSData *)deviceToken
+{
+    NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
+    NSString *formerChannelKey = @"CourierChannel";
+
+    NSString *channel = [[NSTimeZone localTimeZone] name];
+
+    NSString *formerChannel = [userDefaults stringForKey:formerChannelKey];
+    if (formerChannel && ![channel isEqualToString:formerChannel]) {
```

Comment:
What do you think about reversing this comparison? That way, if `formerChannel` is `nil` the expression would be a `nil` message-send, and evaluate to false, meaning that you don't need to check for `nil` first.